src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -391,7 +391,7 @@ static inline int MPIDI_OFI_init_generic(int rank,
         /* -------------------------------- */
         /* Create our address table from    */
         /* encoded KVS values               */
-        /* -------------------------------- */
+        ui/* -------------------------------- */
```

Comment:
I'm still seeing build failures. This is an example of one line that's broken.

This is our [m.txt](https://github.com/pmodels/mpich/files/586021/m.txt).